### ISSUE TITLE
English version

### ISSUE THREAD

**Issue by luravoid** [guopenghui/obsidian-language-learner#6]
Hi, are you planning to release an english version of this plugin? The plugin looks great so it would be really cool if you could

**Comment by luravoid** [comment]
Yes, monolingual dictionaries are very helpful in learning languages ​​once you have reached a certain level of understanding. English is also my second language and I use a monolingual dictionary 99% of the time. The best dictionaries I used are Cambridge and Merriam-Webster

**Comment by shunnkou** [comment]
Hello! Would it be possible to implement an English-Chinese dictionary as well?

**Comment by DaTouCaoSir** [comment]
> It's not difficult. The only problem is the dictionary. Because my first language is Chinese, there is only one Chinese-English dictionary(Youdao dictionary) in this plugin.
> 
> I'm not sure if an English-English dictionary is helpful to people of other languages, and do you have any suggestion for dictionary?

感谢作者制作了这么出色的插件！同上，也希望作者考虑提供自定义英英词典的选择，个人感觉有一定基础后，查找学习单词含义用英英词典更有帮助。再次感谢您的付出！

**Comment by nerd010** [comment]
如果英文原句中包含数字的话，无法选中整句进行翻译

**Comment by DawnEveTiger** [comment]
> It's not difficult. The only problem is the dictionary. Because my first language is Chinese, there is only one Chinese-English dictionary(Youdao dictionary) in this plugin.
> 
> I'm not sure if an English-English dictionary is helpful to people of other languages, and do you have any suggestion for dictionary?

其实用英英字典学单词比中英字典效率高，如果可以的话请考虑一下添加英英字典，[掌上百科](https://www.pdawiki.com/forum/)有现成的词典库，比较常用的有vocabulary.com，urban dictionary，Oxford English Living Dictionary等。非常感谢作者大大的付出！

**Comment by DawnEveTiger** [comment]
> 

在学习日常用品名词的时候是这样的，但更多单词在英文语意里有复杂的意思，如果全用中文记的话会非常痛苦，并且无法摈弃中英互换的输入输出情况。无论是书面英语还是口头英语，最理想的状态都是英英输入，就拿【dam】这个单词来说，假如交流中不记得【堤坝】的英文单词，那么我们可以用解释法【a wall built across a river that stops the river's flow and collects the water】，对方立马秒懂你的意思了，这在雅思口语中也是英文能力的体现之一

**Comment by DawnEveTiger** [comment]
> @DawnEveTiger 现在有道里面的柯林斯部分以及剑桥词典，其实都有英英释义。或者说英英词典是指必须只有英语的词典吗。
> 
> 我感觉只看英文释义描述有时候和猜谜一样，还要费劲猜单词的意思。比如`dam`，英文释义是_a wall built across a river that stops the river's flow and collects the water, especially to make a reservoir (= an artificial lake) that provides water for an area_，看这么长的释义，没有直接看_水坝_两个字效率高。 我觉得英文释义是在对某个词的内涵需要精确把握时比较有效。中文释义加英文描述同时看，效率是最高的。

> 在学习日常用品名词的时候是这样的，但更多单词在英文语意里有复杂的意思，如果全用中文记的话会非常痛苦，并且无法摈弃中英互换的输入输出情况。无论是书面英语还是口头英语，最理想的状态都是英英输入输出。就拿【dam】这个单词来说，假如交流中不记得【堤坝】的英文单词，那么我们可以用解释法【a wall built across a river that stops the river's flow and collects the water】，对方立马秒懂你的意思了，这在雅思口语中也是英文能力的体现之一。在英语使用环境里，【dam】这个单词有着比【堤坝】更多的意思，比如1. the misspellling of damn; 2. The female parent of an animal, especially a domestic mammal. 这些意思看英文一秒领会，特意找中文的话太耗费时间，并且无法做到准确及时输出。

**Comment by SalahAdDin** [comment]
Please!
